apps/flows/src/app/transfer-and-pay-flow/transfer-and-pay-modal/transfer-and-pay-modal.component.html

```diff
@@ -0,0 +1,89 @@
+<kirby-page-title>Transfer and pay</kirby-page-title>
+
+<flows-transfer-registered
+  *ngIf="transferRegistered"
+  [verifiedDetails]="verifiedDetails"
+></flows-transfer-registered>
+<div *ngIf="!transferRegistered">
+  <input
+    class="currency-input currency-input_large-text"
+    maxlength="14"
+    type="text"
+    #input
+    [(ngModel)]="currency"
+    (input)="onCurrencyEntered()"
+  />
+  <span class="currency-input_small-text">Kr.</span>
+</div>
+
+<kirby-modal-footer *ngIf="transferRegistered === false">
+  <div class="container-over" *ngIf="receiverChosen === false">
+    <div class="grid" (click)="showModalChooseOwnAccount()">
+      <p>From</p>
+      <span *ngFor="let account of selectedAccount"
+        ><b>{{ account.name }}</b></span
+      >
+    </div>
+    <kirby-divider></kirby-divider>
+    <div class="container-under">
+      <button kirby-button (click)="showModalChooseReciever()">Choose receiver</button>
+    </div>
+  </div>
+
+  <div class="container-over" *ngIf="receiverChosen === true">
+    <div class="grid" (click)="showModalChooseOwnAccount()">
+      <p>From</p>
+      <span *ngFor="let account of selectedAccount"
+        ><b>{{ account.name }}</b></span
+      >
+    </div>
+    <kirby-divider></kirby-divider>
+    <div class="grid" (click)="showModalChooseReciever()">
+      <p>To</p>
+      <span *ngFor="let other of selectedReceiver"
+        ><b>{{ other.name }}</b></span
+      >
+    </div>
+    <kirby-divider></kirby-divider>
+    <div class="grid" (click)="showModalChooseDate()">
+      <p>Date</p>
+      <p>
+        <b>{{ date }}</b>
+      </p>
+    </div>
+    <kirby-divider></kirby-divider>
+    <div class="grid" (click)="showModalChooseTextAndMessage()">
+      <p>Text</p>
+      <p>
+        <b>{{ text }}</b>
+      </p>
+    </div>
+    <kirby-divider></kirby-divider>
+    <div class="grid" (click)="showModalChooseTextAndMessage()">
+      <p>Message</p>
+      <p>
+        <b>{{ message }}</b>
+      </p>
+    </div>
+
+    <div class="container-under">
+      <button
+        kirby-button
+        [attr.disabled]="!currencyEntered ? true : null"
+        (click)="
+          showModalVerifyDetails(
+            false,
+            currency,
+            selectedReceiver[0].name,
+            selectedAccount[0].name,
+            text,
+            message,
+            date
+          )
+        "
```

Comment:
write a function which calls `showModalVerifyDetails` with the arguments you want rather than doing it here

Reply:
fixed